deny.toml

```diff
@@ -13,6 +13,7 @@ unlicensed = "deny"
 allow = [
     "Apache-2.0",
     "MIT",
+    "Unicode-DFS-2016",
```

Comment:
I have no idea what this thing is @buffet, you're going to have to explain it to me later.